core/domain/wipeout_service_test.py

```diff
@@ -642,7 +642,8 @@ def test_run_user_deletion_completion_user_wrongly_deleted_emails_disabled(
             email_manager,
             'send_mail_to_admin',
             lambda x, y: None,
-            called=False  # Func shouldn't be called when emails are disabled.
+            # Func shouldn't be called when emails are disabled.
```

Comment:
This and similar change are done because I introduced a new lint check to prevent trailing/inline comments unless the reason is a valid allowed pragma.